doc/sphinx/practical-tools/coqide.rst

```diff
@@ -321,34 +342,43 @@ The top of the *Shortcuts* section lets you select the allowed modifiers
 that can be selected for the listed menus.  (The changes won't appear until
 you close and reopen the Preferences dialog.)
 
-*Misc*
+*Misc* – to be documented
```

Comment:
```suggestion
*Misc*
```

"to be documented" not really needed; no one will be confused.

Reply:
This single word alone is confusing for me…